cloudca/resource_cloudca_instance.go

```diff
@@ -95,13 +95,6 @@ func resourceCloudcaInstance() *schema.Resource {
 				Computed:    true,
 				Description: "The instance's memory in MB. If the compute offering is custom, this value is required",
 			},
-
-			"purge": &schema.Schema{
-				Type:        schema.TypeBool,
```

Comment:
> // When Removed is set, this attribute has been removed from the schema
    //
    // Removed attributes can be left in the Schema to generate informative error
    // messages for the user when they show up in resource configurations.
    // This string is the message shown to the user with instructions on
    // what do to about the removed attribute.
    Removed string